core/src/main/java/com/oath/oak/EntrySet.java

```diff
@@ -965,19 +727,7 @@ boolean isEntryDeleted(int ei) {
      * by other thread just after this check. For the thread safety use a copy of value reference (next method.)
      * */
     boolean isValueRefValid(int ei) {
-        return getValueReference(ei) != INVALID_VALUE_REFERENCE;
-    }
-
-    /*
-    * When value is connected to entry, first the value reference is CASed to the new one and after
-    * the value version is set to the new one (written off-heap). Inside entry, when value reference
-    * is invalid its version can only be invalid (0) or negative. When value reference is valid and
-    * its version is either invalid (0) or negative, the insertion or deletion of the entry wasn't
-    * accomplished, and needs to be accomplished.
-    * */
-    boolean isValueLinkFinished(LookUp lookUp) {
```

Comment:
Was it removed because new state in the ThreadContext has this info? But I didn't see this new state of linkage being incomplete..

Reply:
This method was not removed, it was moved to ThreadContext.
This comment could have been avoided if you would have reviewed this PR in the order I suggested.